students/john_rogers/lesson09/mailroom.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+"""
+mailroom.py: use classes where applicable
+Author: JohnR
+Version: 1.0
+Last updated: 2/17/19
+Notes:
+"""
+
+from donors import DonorDataBase as DDB
+from donors import Donor as D
```

Comment:
avoid aliases unless there is a good reason - _especially_ single letter variables.